### DISCUSSION TITLE
KEEPER 홈페이지를 띄워보자~!

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#141] [1. 개발 관련 글 모음]
https://stump-cress-2f5.notion.site/KEEPER-65f8fad382204aebaa8067a9d255fa64?pvs=4

안녕하세요. 처음 글 올려보네요..! 잘부탁드리겠습니다...!!

**Comment by shkisme** [comment]
공부를 많이 하신 흔적이 글에서 보입니다 잘 읽었어요 👍👍

**Reply by 02ggang9** [reply]
감사합니다~~!!

**Comment by pipisebastian** [comment]
우왕! 엄청 꼼꼼하게 정리 하셨네요! 알아갑니당~~

**Reply by pipisebastian** [reply]
도커 프로세스 격리 원리에 관해서 [블로그](https://velog.io/@pipi/docker-%EB%8F%84%EC%BB%A4-%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC)에 정리해봤었는데,, 남기고 가겠습니다 ㅎㅎ..

**Reply by 02ggang9** [reply]
헐.. 감사합니다~~ 도커랑 쿠버 네트워크 때문에 머리 아팠는데 잘 참고하도록 하겠습니닷!!
